tests/utils/suite_writer/slash_run_result.py

```diff
@@ -111,7 +111,7 @@ def __getitem__(self, args):
 
     def __contains__(self, args):
         return self._normalize_key(args) in self._events_by_args
-        return self.has_event(*args)
+        # return self.has_event(*args)
```

Comment:
It fails on rather (on not) an event exists. Assertions like:
```
assert not summary.events.has_event('NEVER')
assert evt1 in summary.events
assert cleanup in summary.events
```
(24 tests failed once I change this method to return `self.has_event(*args)`